packages/typespec-test/test/todo_non_branded/generated/typespec-ts/review/todo-non-branded.api.md

```diff
@@ -4,605 +4,239 @@
-    status: "200";
+    contentType?: string;
 }
 
 // @public
-export interface TodoItemsGet404Response extends HttpResponse {
-    // (undocumented)
-    status: "404";
-}
-
-// @public (undocumented)
-export type TodoItemsGetParameters = RequestParameters;
-
-// @public (undocumented)
-export interface TodoItemsList {
-    // (undocumented)
-    get(options: TodoItemsListParameters): StreamableMethod<TodoItemsList200Response>;
-    // (undocumented)
-    post(options: TodoItemsCreateJsonParameters): StreamableMethod<TodoItemsCreateJson200Response | TodoItemsCreateJson422Response>;
-    // (undocumented)
-    post(options: TodoItemsCreateFormParameters): StreamableMethod<TodoItemsCreateForm200Response | TodoItemsCreateForm422Response>;
+export interface TodoItemsDeleteOptionalParams extends OperationOptions {
 }
 
 // @public
-export interface TodoItemsList200Response extends HttpResponse {
-    // (undocumented)
-    body: TodoPageOutput;
-    // (undocumented)
-    status: "200";
-}
-
-// @public (undocumented)
-export type TodoItemsListParameters = TodoItemsListQueryParam & RequestParameters;
-
-// @public (undocumented)
-export interface TodoItemsListQueryParam {
-    // (undocumented)
-    queryParameters: TodoItemsListQueryParamProperties;
+export interface TodoItemsGetOptionalParams extends OperationOptions {
 }
 
-// @public (undocumented)
-export interface TodoItemsListQueryParamProperties {
-    limit: number;
-    offset: number;
+// @public
+export interface TodoItemsListOptionalParams extends OperationOptions {
+    limit?: number;
+    offset?: number;
 }
 
 // @public
-export interface TodoItemsUpdate200Response extends HttpResponse {
+export interface TodoItemsOperations {
     // (undocumented)
-    body: TodoItemOutput;
+    attachments: TodoItemsAttachmentsOperations;
     // (undocumented)
-    status: "200";
-}
-
-// @public (undocumented)
-export interface TodoItemsUpdateBodyParam {
+    create: (item: TodoItem, options?: TodoItemsCreateOptionalParams) => Promise<{
+        id: number;
+        title: string;
+        createdBy: number;
+        assignedTo?: number;
+        description?: string;
+        status: "NotStarted" | "InProgress" | "Completed";
+        createdAt: Date;
+        updatedAt: Date;
+        completedAt?: Date;
+        labels?: TodoLabels;
+    }>;
+    delete: (id: number, options?: TodoItemsDeleteOptionalParams) => Promise<void>;
+    // (undocumented)
+    get: (id: number, options?: TodoItemsGetOptionalParams) => Promise<{
+        id: number;
+        title: string;
+        createdBy: number;
+        assignedTo?: number;
+        description?: string;
+        status: "NotStarted" | "InProgress" | "Completed";
+        createdAt: Date;
+        updatedAt: Date;
+        completedAt?: Date;
+        labels?: TodoLabels;
+    }>;
     // (undocumented)
-    body: {
-        patch: TodoItemPatch;
-    };
+    list: (options?: TodoItemsListOptionalParams) => Promise<TodoPage>;
```

Comment:
No pagination support (presumably we don't support the new TypeSpec paging yet)

Reply:
We need to support this in unbranded client: https://github.com/Azure/autorest.typescript/issues/2932.

Reply:
Should be covered by sdkmethod adoption pr.